src/sentry/discover/endpoints/discover_default_query.py

```diff
@@ -0,0 +1,107 @@
+from rest_framework import status
+from rest_framework.exceptions import ParseError
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.base import pending_silo_endpoint
+from sentry.api.bases import NoProjects, OrganizationEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.discover.endpoints.bases import DiscoverSavedQueryPermission
+from sentry.discover.endpoints.serializers import DiscoverSavedQuerySerializer
+from sentry.discover.models import DiscoverSavedQuery
+
+
+def get_default_query(organization, user):
+    return DiscoverSavedQuery.objects.get(
+        organization=organization, is_default=True, created_by=user
```

Comment:
If default queries are per-org won't including the user condition mean that only one user in an org can use the default?

Reply:
So this is actually intended to be a per-user default. We have plans to make the landing page for Discover default to a customized query since different users have different jumping off points and All Events might be including more than they need

For us, we typically dump a lot of the columns so if we could have that state saved, it would lower some overhead

Reply:
Right, but the index that is being added in #39063 doesn't include user, so only one query per-org can be `is_default` and the chances of the current user being the 'winner' gets low in large orgs.

Reply:
Resolving since I fixed it in the uniqueness constraint here: https://github.com/getsentry/sentry/pull/39063#discussion_r978965723